api/files/serializers.py

```diff
@@ -442,6 +442,16 @@ def to_representation(self, value):
         if guid:
             data['data']['id'] = guid._id
 
+        if isinstance(value.target, Registration) and value.target.is_retracted:
```

Comment:
Is this actually reachable? Shouldn't the view 410 before any data hits the serializer?